numpy/core/src/multiarray/multiarraymodule.c

```diff
@@ -4767,12 +4773,12 @@ PyMODINIT_FUNC PyInit__multiarray_umath(void) {
     if (initumath(m) != 0) {
         goto err;
     }
-    return m;
+    return HPy_FromPyObject(ctx, m);
```

Comment:
ah, this is interesting. I always assumed that the first thing to do to port a module to HPy would be to convert `PyModuleDef` into `HPyModuleDef` and thus using `HPyModule_Create`, but indeed, it's not required.

It is worth adding a note to `docs/porting-guide.rst`, maybe showing an example of the minimal diff which is required to hpy-ify a module.